To enable this patch, you will need to edit the PATCHES-* variable inside
the Makefile.

Instead of patching this change can also be done by the OS. Patching offers
some advantages, though:
- the changed layout is available in BIOS, bootloader, ...
- remote connections - both incoming and outgoing - tend to have very
  weird keyboard mapping problems. This reduces those to what you'd face
  with a US layout, which contains a lot less changed keys.

--- .prepatched.hex	2020-06-29 15:35:02.532207474 +0300
+++ .dvorak_caps.hex	2020-06-29 15:35:03.323227821 +0300
@@ -8431,14 +8431,14 @@
 00021370  58 00 59 6f 00 00 00 00  9e 9e 00 9f 9f 00 4a 4a  |X.Yo..........JJ|
 00021380  00 00 00 00 b2 00 00 00  7c 11 02 00 40 00 00 00  |........|...@...|
 00021390  20 13 02 00 e0 12 02 00  0b 00 00 00 60 13 02 00  | ...........`...|
-000213a0  01 02 11 10 1f 6e 2e 83  00 00 00 00 00 00 00 00  |.....n..........|
-000213b0  70 03 12 1e 20 2d 2f 00  00 00 00 00 00 00 00 00  |p... -/.........|
-000213c0  71 04 13 72 21 73 30 00  00 00 00 00 00 00 00 00  |q..r!s0.........|
-000213d0  06 05 14 15 22 23 31 32  00 00 00 00 00 00 00 00  |...."#12........|
-000213e0  07 08 17 16 25 24 34 33  00 00 00 00 00 00 00 00  |....%$43........|
-000213f0  0d 09 18 1c 26 75 35 38  00 00 00 00 00 00 00 00  |....&u58........|
-00021400  77 0a 19 76 27 84 36 85  00 00 00 00 00 00 00 00  |w..v'.6.........|
-00021410  0c 0b 1a 1b 28 29 2a 37  00 00 00 00 00 00 00 00  |....()*7........|
+000213a0  01 02 29 10 1f 6e 28 83  00 00 00 00 00 00 00 00  |..)..n(.........|
+000213b0  70 03 35 3a 19 2d 11 00  00 00 00 00 00 00 00 00  |p.5:.-..........|
+000213c0  71 04 36 72 13 73 25 00  00 00 00 00 00 00 00 00  |q.6r.s%.........|
+000213d0  06 05 1a 16 17 18 26 2f  00 00 00 00 00 00 00 00  |......&/........|
+000213e0  07 08 23 22 24 21 34 32  00 00 00 00 00 00 00 00  |..#"$!42........|
+000213f0  1c 09 30 0d 15 75 12 38  00 00 00 00 00 00 00 00  |..0..u.8........|
+00021400  77 0a 14 76 33 84 31 85  00 00 00 00 00 00 00 00  |w..v3.1.........|
+00021410  1b 0b 27 37 20 0c 2a 2e  00 00 00 00 00 00 00 00  |..'7 .*.........|
 00021420  78 79 0e 0f 1d 74 2b 3d  00 00 00 00 00 00 00 00  |xy...t+=........|
 00021430  4b 7b 00 9b 00 00 00 59  00 00 00 00 00 00 00 00  |K{.....Y........|
 00021440  4c 7a 9a 99 98 97 a0 54  00 00 00 00 00 00 00 00  |Lz.....T........|
@@ -8446,8 +8446,8 @@
 00021460  50 51 00 00 00 53 7e 4f  00 00 00 00 00 00 00 00  |PQ...S~O........|
 00021470  00 7c 7d 00 00 3c 00 3e  00 00 00 00 00 00 00 00  |.|}..<.>........|
 00021480  00 00 00 2c 00 00 39 00  00 00 00 00 00 00 00 00  |...,..9.........|
-00021490  3a 00 00 00 00 00 40 00  00 00 00 00 00 00 00 96  |:.....@.........|
-000214a0  00 9d 00 9e 9f 4a 3a 9c  7d 7e 00 00 00 00 00 00  |.....J:.}~......|
+00021490  1e 00 00 00 00 00 40 00  00 00 00 00 00 00 00 96  |......@.........|
+000214a0  00 9d 00 9e 9f 4a 1e 9c  7d 7e 00 00 00 00 00 00  |.....J..}~......|
 000214b0  ff 00 7f 00 7f 00 ff 00  ff 00 ff 00 ff 00 ff 00  |................|
 000214c0  ff 00 8b 00 ff 00 d3 00  e3 00 a6 00 48 00 41 80  |............H.A.|
 000214d0  fa 03 00 00 a0 97 00 00  10 01 00 00 a0 13 02 00  |................|
